canonical/vanilla-framework
# Update icon SVGs to use simplified paths and fix artifacts

## Done

Updates certification and arrow icons SVGs to fix path issues.

## QA

- Open [demo](https://vanilla-framework-5385.demos.haus/docs/examples/patterns/icons/icons-additional?theme=light)
- Make sure arrow and certification icons look ok

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix release (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="268" alt="image" src="https://github.com/user-attachments/assets/d0e6897b-19f6-474a-a4fc-bfa65b807dc3">
